WIP: finish l2_service.py; add addtional network_helper methods.

jlongstaf opened the pull request:
WIP -- finish l2_service.py. Added additonal methods to network_helper.

jlongstaf committed:
WIP: finish l2_service.py; add addtional network_helper methods.

jlongstaf committed:
Fixed errors found in code review.

richbrowne commented on f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py:
```diff
@@ -359,11 +359,25 @@ def arp_delete_by_mac(self,
+                   bigip,
+                   ip_address,
+                   partition=const.DEFAULT_PARTITION):
+        if ip_address:
```
Add note about the 2 implementations in lbaasv1 common code.

richbrowne commented on f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py:
```diff
@@ -461,20 +475,131 @@ def add_fdb_entry(
+        fdb_entry['name'] = mac_address
+        fdb_entry['endpoint'] = vtep_ip_address
+
+        for i in range(len(records)):
```
Could use:

i, record in enumerate(records)

jlongstaf committed:
Fixed additional issues from code review: fix indent error
in network_sevice; fix partion name, create call, missing
bigip param in selfips.py
